node/rpc/src/auth_api.rs

```diff
@@ -18,6 +20,7 @@ where
     B: Beacon + Send + Sync + 'static,
 {
     let (perms,) = params;
+    info!("Auth new requested for permissions: {:?}", perms);
```

Comment:
I'd remove these logs. They are already captured by the logs that print when we call an RPC method